src/extensions/default/Inline3DParametersEditor/ParameterEditor.js

```diff
@@ -69,8 +69,7 @@ define(function (require, exports, module) {
     };
 
     ParameterEditor.prototype._setSpaces = function (parameters) {
-        this._spaces = [];
-        this._spaces.push(parameters.search(/\S/));
+        this._spaces = [parameters.search(/\S/)];
```

Comment:
Hm, based on that explanation, it very much looks like that is not what this line of code does. For instance, when `parameters` is the string `"2 3 3"` then the result of '[parameters.search(/\S/)]' is the array `[0]`, because `search` is effectively an `indexOf` for regular expressions, and the first hit for "not whitespace" (the `/\S/` regexp) is string position 0.

Can you explain how "2 3 3" turns into `[5, 3,  2, 0]`, etc.?